src/main/scala/org/allenai/plugins/NodeJsPlugin.scala

```diff
@@ -213,8 +236,8 @@ object NodeJsPlugin extends AutoPlugin {
   /** Execute `cmd` with `npm` setting `root` as the working directory.
     * @throws Exception if the process returns an non-zero exit code
     */
-  private def exec(cmd: String, root: File, env: Map[String, String]): Unit = {
-    fork(cmd, root, env).exitValue() match {
+  private def exec(cmd: String, root: File, env: Map[String, String], npmLogLevel: String): Unit = {
+    fork(s"$cmd --loglevel $npmLogLevel", root, env).exitValue() match {
```

Comment:
Validate the log level? I wouldn't want to cause a weird build failure because of a misconfigured setting . . .